tests/displayAllGenresTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+// use GameCollection\Entities\Game;
+use GameCollection\Models\GameModel;
+use PHPUnit\Framework\TestCase;
+
+require_once 'vendor/autoload.php';
+require 'src/displayAllGenres.php';
+
+class displayAllGenresTest extends TestCase
+{
+    // public function test_success_DisplayGenres()
+    // {
+    //     $input = [];
```

Comment:
This array needs to replicate what your genremodel returns - I'd suggets putting in a var\_dump when you call that function else where, and they trying to replicate the array it gives you in this test